domain/src/main/java/com/jakefoundation/buildabotworkshop/domain/Owner.java

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
this file also needs package name. `com.jakefoundation.buildabotworkshop.domain`

Reply:
okay